Analysis/IsolatedLeptonFinder/src/IsolatedLeptonFinderProcessor.cc

```diff
@@ -423,6 +423,14 @@ ReconstructedParticleImpl* IsolatedLeptonFinderProcessor::CopyReconstructedParti
 	pfo->setParticleIDUsed(pfo_orig->getParticleIDUsed());
 	pfo->setGoodnessOfPID(pfo_orig->getGoodnessOfPID());
 	pfo->setStartVertex(pfo_orig->getStartVertex());
+	for(unsigned int i=0;i<pfo->getTracks().size();i++){
+	  pfo->addTrack(pfo_orig->getTracks()[i]);
+	  std::cout<<" i get into add track "<<std::endl;
```

Comment:
Please use `streamlog_out(DEBUG)` here instead of cout to avoid log pollution

Reply:
Or just drop it